wolox-books/src/views/Register.vue

```diff
@@ -76,24 +78,20 @@ export default {
     }
   },
   methods: {
-    signIn () {
+    submit () {
       if (this.$v.$invalid) {
         this.showErrors = true
       } else {
-        console.log(
-          `
-        {
-          'user': {
-            'email': ${this.email},
-            'password': ${this.password},
-            'password_confirmation': ${this.password},
-            'first_name': ${this.firstName},
-            'last_name': ${this.lastName},
-            'locale': 'en'
+        Register({
+          user: {
+            email: this.email,
```

Comment:
use a const to define all variables as 'this' and you can remove all the 'this.'.

e.g.:
const { email, password } = this
register({ user: { email, password } })